### PR TITLE
[DOCS]chore: optimisation

### PR DESCRIPTION
**Pull request by sstraatemans**

### CODE REVIEW

File: packages/apps/docs/src/components/Layout/Full/components/Aside/AsideList.tsx [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
-    asideListClass,
-    asideListInnerVariants[inner ? 'true' : 'false'],
-  );
-  return (
-    <ul ref={ref} className={classes}>
-      {children}
-    </ul>
-  );
-};
+export const AsideList = React.forwardRef<HTMLUListElement, IProps>(
```
Comment (sstraatemans) [inline packages/apps/docs/src/components/Layout/Full/components/Aside/AsideList.tsx:12]: fixes a console warning

File: packages/apps/docs/.storybook/preview.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { darkThemeClass } from '@kadena/react-ui/theme';
 
 import { themes } from '@storybook/theming';
-import { RouterContext } from 'next/dist/shared/lib/router-context'; // next 12
+import { RouterContext } from 'next/dist/shared/lib/router-context.shared-runtime'; // next 13
```
Comment (sstraatemans) [inline packages/apps/docs/.storybook/preview.ts:4]: needed to fix the build after next update

File: packages/apps/docs/src/utils/staticGeneration/checkSubTreeForActive.mjs [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { getData } from './getData.mjs';
 import path from 'path';
 
-const omit = (obj, ...keysToOmit) =>
+const omit = (obj, keysToOmit) =>
```
Comment (sstraatemans) [inline packages/apps/docs/src/utils/staticGeneration/checkSubTreeForActive.mjs:4]: this makes sure we need a lot less data in the getstaticprops.
making the website much faster

File: packages/apps/docs/src/components/BottomPageSection/BottomPageSection.tsx [hunk 4]
```diff
@@ -68,7 +68,11 @@ export const BottomPageSection: FC<IProps> = ({
         )}
       </Stack>
       <Divider />
-      <Stack justifyContent="space-between" width="100%">
+      <Stack
+        direction={{ xs: 'column', lg: 'row' }}
```
Comment (sstraatemans) [inline packages/apps/docs/src/components/BottomPageSection/BottomPageSection.tsx:72]: aligning the subscribe and the helpful component on mobile

File: packages/apps/immutable-records/package.json [hunk 5]
```diff
@@ -20,6 +20,7 @@
   "dependencies": {
     "@kadena/react-ui": "workspace:*",
     "@vanilla-extract/css": "1.12.0",
+    "mini-css-extract-plugin": "2.7.6",
```
Comment (sstraatemans) [inline packages/apps/immutable-records/package.json:23]: for some reason this was not in this app. and it kills the build

File: packages/apps/docs/next.config.mjs [hunk 6]
```diff
@@ -92,8 +92,11 @@ const nextConfig = {
   eslint: {
     ignoreDuringBuilds: true, // lint is a different task/phase
   },
+
   pageExtensions: ['ts', 'tsx', 'js', 'jsx', 'md', 'mdx'],
   reactStrictMode: true,
+  productionBrowserSourceMaps: true,
```
Comment (realdreamer) [inline packages/apps/docs/next.config.mjs:98]: why are we enabling sourcemaps for prod?

File: packages/apps/docs/src/pages/_app.tsx [hunk 7]
```diff
@@ -60,6 +59,7 @@ export const MyApp = ({
         <title>{props.frontmatter.title}</title>
         <meta name="title" content={props.frontmatter.title} />
         <meta name="description" content={props.frontmatter.description} />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (realdreamer) [inline packages/apps/docs/src/pages/_app.tsx:62]: Good catch 👍🏽

**Review by realdreamer — APPROVED** [review]

**Pull request merged**